teachbooks/serve.py

```diff
@@ -46,7 +46,12 @@ def __init__(self, servedir: Path | str, workdir: Path | str, port: int | None =
             old_instance = self.load(workdir)
             self.__dict__ = old_instance.__dict__
         except ServerError:
-            self.servedir = Path(servedir)
+            # Check is arguments were given
+            if len(sys.argv) == 2:
```

Comment:
i don't understand where these arguments would come from, nor why the second would be the server directory.

Reply:
I import sys. And if you import sys then you can reach argv directly